azure-pipelines.yml

```diff
@@ -32,27 +32,27 @@ jobs:
       sudo apt-get install -y dotnet-sdk-6.0
     displayName: install .Net
   - script: |
-      sudo apt-get install -y \
-          libboost1.71-dev \
-          libboost-program-options1.71-dev \
-          libboost-system1.71-dev \
-          libboost-thread1.71-dev \
-          libboost-atomic1.71-dev \
-          libboost-chrono1.71-dev \
-          libboost-container1.71-dev \
-          libboost-context1.71-dev \
-          libboost-contract1.71-dev \
-          libboost-coroutine1.71-dev \
-          libboost-date-time1.71-dev \
-          libboost-fiber1.71-dev \
-          libboost-filesystem1.71-dev \
-          libboost-graph-parallel1.71-dev \
-          libboost-log1.71-dev \
-          libboost-regex1.71-dev \
+      sudo apt-get install -y --fix-broken\
+          libboost-dev \
+          libboost-program-options-dev \
+          libboost-system-dev \
+          libboost-thread-dev \
+          libboost-atomic-dev \
+          libboost-chrono-dev \
+          libboost-container-dev \
+          libboost-context-dev \
+          libboost-contract-dev \
+          libboost-coroutine-dev \
+          libboost-date-time-dev \
+          libboost-fiber-dev \
+          libboost-filesystem-dev \
+          libboost-graph-parallel-dev \
+          libboost-log-dev \
+          libboost-regex-dev \
           googletest \
           libgtest-dev \
+          libgcc-10-dev \
```

Comment:
This shouldn't need to be explicitly specified; this gets automatically installed with the compiler.

Reply:
Hi Sai, thanks for the tip! 

I have addressed above comments.